src/viam/sdk/tests/test_camera.cpp

```diff
@@ -28,248 +28,70 @@ using namespace viam::sdk;
 
 BOOST_AUTO_TEST_SUITE(test_camera)
 
-std::shared_ptr<MockCamera> camera = MockCamera::get_mock_camera();
-
 BOOST_AUTO_TEST_CASE(test_get_image) {
-    Camera::raw_image expected_image = fake_raw_image();
-    Camera::raw_image image = camera->get_image("JPEG", {});
+    std::shared_ptr<MockCamera> mock = MockCamera::get_mock_camera();
+    client_to_mock_pipeline<CameraClient>(mock, [&](Camera& client) {
```

Comment:
does this lambda need to capture by ref? I don't see where we're using `mock` anywhere in the test. Same is true for several tests below.

Reply:
It does not; good eye! Did not capture by ref here and below.